SiteLogToGeodesyML/modified-schemas/geodeticMonument.xsd

```diff
@@ -121,7 +120,9 @@ Copyright: Scripps Orbit and Permanent Array Center (SOPAC), Commonwealth Govern
                     <!-- Height (of the monument, in meters) -->
                     <element minOccurs="0" name="monumentHeight">
                         <simpleType>
-                            <restriction base="geo:NillableDouble">
+                            <restriction base="decimal">
+                                <totalDigits value="6"/>
+                                <fractionDigits value="4"/>
```

Comment:
Is this change introduced by version 0.5 of the schema, or will it restore some modification made to the modified-schemas?

Reply:
Yes, it was introduced by version 0.5 of the schema. We did not manually made this change.

Reply:
Good.